RecoLocalMuon/GEMRecHit/python/gemRecHits_cfi.py

```diff
@@ -12,7 +12,9 @@
 
 
 from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+from Configuration.Eras.Modifier_run3_GEM_2025_cff import run3_GEM_2025
 from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
 
-run3_GEM.toModify(gemRecHits, ge21Off=True)
-phase2_GEM.toModify(gemRecHits, ge21Off=False)
+run3_GEM.toModify(gemRecHits, ge21Off=False, applyMasking=False)
+run3_GEM_2025.toModify(gemRecHits, gem21Off=False, applyMasking=True)
```

Comment:
There was a typo, indeed:
```suggestion
run3_GEM_2025.toModify(gemRecHits, ge21Off=False, applyMasking=True)
```

Reply:
I think also the ge21Off value should also be True for run3, as it was before